FamilySearchIosSampleApp/TreeTVC.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  TreeVC.swift
+//  FamilySearchIosSampleApp
+//
+//  Created by Eduardo Flores on 6/6/16.
+//  Copyright © 2016 FamilySearch. All rights reserved.
+//
+
+import UIKit
+
+class TreeTVC: UITableViewController {
+    
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    
+    var user : User!
```

Comment:
Implicitly unwrapped optionals are ok for `IBOutlet` properties but shouldn't be used for normal properties.  Just use a normal optional.